skimage/io/_plugins/matplotlib_plugin.py

```diff
@@ -169,10 +169,33 @@ def imshow_collection(ic, *args, **kwargs):
     """Display all images in the collection.
 
     """
-    fig, axes = plt.subplots(1, len(ic))
+    N = len(ic)
+    print N
+    k = (N * 12)**(0.5)
+    r1 = int(k/4)
+    r2 = r1 + 1
+    if N % r1 == 0:
+        c1 = N/r1
+    else:
+        c1 = N/r1 + 1
```

Comment:
That's an interesting idea (one liner). I actually wanted to know if I could define a new function for such cases, and if so in which file?